content/pop-shell.md

```diff
@@ -23,16 +23,25 @@ sudo apt install git node-typescript make gnome-shell-extension-prefs
 
 **Note:** these instructions assume an Ubuntu-based distro; other distributions (see examples below) will use different package managers, and may have different package names.
 
-Next we'll clone the repository using this command:
+Next, we'll clone the repository and enter the directory using these commands:
 
 ```bash
 git clone https://github.com/pop-os/shell.git
+cd shell
 ```
 
-Now we'll enter the directory for <u>Pop Shell</u> (called "shell") and build the package:
+Then, check out the appropriate branch for your version of Ubuntu. Not all Ubuntu codenames are listed; if your version isn't listed, use the closest version _before_ it in alphabetical order.
+
+```
```

Comment:
Do we want to add bash like the other command sections in this article and others?

Reply:
Done.